libecs/CMakeLists.txt

```diff
@@ -31,6 +31,47 @@ fetch_boost_library(typeof)
 fetch_boost_library(utility)
 fetch_boost_library(predef)
 fetch_boost_library(io)
+fetch_boost_library(dll)
+
+fetch_boost_library(filesystem)
+fetch_boost_library(atomic)
+fetch_boost_library(align)
+fetch_boost_library(iterator)
+fetch_boost_library(concept_check)
+fetch_boost_library(conversion)
+fetch_boost_library(function)
+fetch_boost_library(bind)
+fetch_boost_library(move)
+fetch_boost_library(smart_ptr)
+fetch_boost_library(spirit)
+fetch_boost_library(array)
+fetch_boost_library(endian)
+fetch_boost_library(foreach)
+fetch_boost_library(optional)
+fetch_boost_library(phoenix)
+fetch_boost_library(pool)
+fetch_boost_library(proto)
+fetch_boost_library(range)
+fetch_boost_library(regex)
+fetch_boost_library(thread)
+fetch_boost_library(chrono)
+fetch_boost_library(ratio)
+fetch_boost_library(rational)
+fetch_boost_library(container)
+fetch_boost_library(date_time)
+fetch_boost_library(numeric_conversion)
+fetch_boost_library(tokenizer)
+fetch_boost_library(exception)
+fetch_boost_library(intrusive)
+fetch_boost_library(algorithm)
+fetch_boost_library(lexical_cast)
+fetch_boost_library(unordered)
+fetch_boost_library(variant)
+fetch_boost_library(variant2)
+fetch_boost_library(system)
+fetch_boost_library(type_index)
+fetch_boost_library(winapi)
+fetch_boost_library(mp11)
```

Comment:
At this point, we should clone the entire boost lib, less nightmare
https://github.com/boostorg/boost